VisualRecognitionV4.playground/Pages/Collections.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,75 @@
+//:## Collections
+
+import Foundation
+import VisualRecognitionV4
+
+let visualRecognition = setupVisualRecognitionV4()
+var createdCollection = "d156c36c-2ac3-499c-a026-fd6e8f8c1687"
```

Comment:
Use this collection id (at least in api definition): `"collection_id": "5826c5ec-6f86-44b1-ab2b-cca6c75f2fc7"`